mk/rt.mk

```diff
@@ -35,8 +35,8 @@
 # that's per-target so you're allowed to conditionally add files based on the
 # target.
 ################################################################################
-NATIVE_LIBS := rust_builtin hoedown morestack miniz \
-		rustrt_native rust_test_helpers
+NATIVE_LIBS := rust_builtin hoedown miniz \
+		rust_test_helpers morestack
```

Comment:
Yeah it's required for stage0 currently (it's just an empty object file), but the stage1+ compilers don't require morestack so this can be removed after a snapshot.